### ISSUE TITLE
Edys Car SImulator

### ISSUE THREAD

**Issue by Olivem1234567** [3kh0/3kh0.github.io#129]
### What is this request?

New game

### Please describe the new feature here

 link : https://unblocked-games.s3.amazonaws.com/edys-car-simulator.html

**Comment by Russell2259** [comment]
Duplicate of #103.

**Comment by Olivem1234567** [comment]
is it added?
